nemo/collections/nlp/modules/common/text_generation_server.py

```diff
@@ -192,7 +199,7 @@ def put(self):
                 repetition_penalty,
                 end_strings=end_strings,
                 min_tokens_to_generate=min_tokens_to_generate,
-                **extra,
+                compute_logprob=compute_logprob ** extra,
```

Comment:
there is a comma in between `compute_logprob=compute_logprob` and `** extra,`

Reply:
remember run `python setup.py style --fix`

Reply:
thanks